docker_test.go

```diff
@@ -1267,3 +1268,31 @@ func TestContainerNonExistentImage(t *testing.T) {
 	})
 
 }
+
+func TestDockerContainerCopyFileToContainer(t *testing.T) {
+	ctx := context.Background()
+
+	nginxC, err := GenericContainer(ctx, GenericContainerRequest{
+		ContainerRequest: ContainerRequest{
+			Image:        "nginx:1.17.6",
+			ExposedPorts: []string{"80/tcp"},
+			WaitingFor:   wait.ForListeningPort("80/tcp"),
+		},
+		Started: true,
+	})
+	defer nginxC.Terminate(ctx)
+
+	fileContent, err := ioutil.ReadFile("./testresources/hello.sh")
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
With above explanation, this would be a matter of the internals of the copy method, simplifying client code's life